nedap/formatting-stack
# Fix linting renamed files

## Brief


Renaming files would yield existing-files errors when using branch-formatter

Using `git diff --diff-filtes` allows us to filter files with a specific status, in our case we only want Added, Copied, Modified and Renamed files. (excludes files which are unmerged, had a typechange, are unknown, have/had broken pairing, and are deleted)

## QA plan

 1. checkout a project with f-s
 1. rename & stage file
 1. run branch-formatter
 1. normal linting as expected


## Author checklist



* Additionally, I have code-reviewed iteratively the PR considering the following aspects in isolation:

## Reviewer checklist

* Additionally, I have code-reviewed iteratively the PR considering the following aspects in isolation:
